### PR TITLE
Add section about being a fork and community developed

### PR DESCRIPTION
**Pull request by MaxLeiter**
Alternative to #39

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,6 +64,10 @@
 				<h3>Open Source</h3>
 				<p>The Lounge is open source and licensed under the MIT license.</p>
 			</div>
+			<div class="col-sm-4 feature">
+				<h3>Community Managed</h3>
+				<p>The Lounge is an active community fork of <a href="http://www.shout-irc.com">Shout</a>. It's development is made possible by a growing and dedicated community. You can find us in #thelounge on Freenode.</p>
```
Comment (astorije) [inline index.html]: - s/`It's`/`Its`/
- s/`You can find us`/`Find us`/ ?

Comment (astorije) [inline index.html]: Actually, this is too big compared to the other blocks, I think:

screen shot 2016-09-19 at 20 37 50

What about:

```html
It is run by a growing and dedicated community as a fork of <a href="http://www.shout-irc.com">Shout</a>. Find us in <code>#thelounge</code> on Freenode.</p>
```

Which gives:

screen shot 2016-09-19 at 20 50 00

Not perfect but I don't have anything better...

**Comment by MaxLeiter** [comment]
If I remove "growing and" it looks a little better, how about that?

**Comment by astorije** [comment]
Yep, go for it!

**Comment by MaxLeiter** [comment]
Should be good now?

**Comment by astorije** [comment]
Argh, sorry, I thought you meant removing this from my suggestion:

```html
It is run by a dedicated community as a fork of <a href="http://www.shout-irc.com">Shout</a>. Find us in <code>#thelounge</code> on Freenode.</p>
```

You still have a typo, and it's still way too long for the "feature blocks" IMO.

**Comment by MaxLeiter** [comment]
Should be good now @astorije. Fingers crossed!

**Pull request merged**